doc/api/worker_threads.md

```diff
@@ -242,6 +268,20 @@ new Worker('process.env.SET_IN_WORKER = "foo"', { eval: true, env: SHARE_ENV })
   });
 ```
 
+## `worker.setPlatformData(value)`
```

Comment:
@jasnell Thank you for working on this.  I think this function would need to take a key and value to do what would be needed by tooling.  The reason is that nyc would need to use this in a way that other modules would not accidentally overwrite the data set by nyc (and vise versa).

I think for nyc to safely use this feature we need `setPlatformData(key, value)`.  In that case nyc would probably use a `__filename` as the key, that way other modules using setPlatformData would be very unlikely to conflict with us.  Once we have the ability to force a `--require` into the worker thread that code would look at `worker_threads.platformData[nycFilename]`.